pkg/kubectl/cmd/config/set.go

```diff
@@ -62,6 +66,8 @@ func NewCmdConfigSet(out io.Writer, configAccess ConfigAccess) *cobra.Command {
 		},
 	}
 
+	f := cmd.Flags().VarPF(&options.setRawBytes, "set-raw-bytes", "", "Write the given PROPERTY_VALUE directly to the binary field rather than base64 decoding first")
```

Comment:
Suggested help string: `When writing a []byte PROPERTY_VALUE, write the given string directly without base64 decoding.`